kbase-extension/static/kbase/js/widgets/narrative_core/upload/stagingAreaViewer.js

```diff
@@ -133,7 +131,7 @@ define([
                 .then((data) => {
                     const files = JSON.parse(data).map((f) => {
                         f.path = f.path.substring(f.path.indexOf('/') + 1);
-                        f.subdir = '/' + f.path.substring(0, f.path.lastIndexOf('/')) || '/';
```

Comment:
the `|| '/'` never gets used as the previous expression always evaluates to true